test/benchgc/src/benchgc/mlir/bench.py

```diff
@@ -0,0 +1,206 @@
+################################################################################
```

Comment:
This file should be placed at the top level, not in `mlir` folder.

Reply:
fixed